manager/service.go

```diff
@@ -185,7 +185,10 @@ func (ms *managerService) Run(ctx context.Context, c *manager.ComputationRunReq)
 		return "", err
 	}
 
-	ms.vms[c.Id].Transition(manager.VmRunning)
+	if err := ms.vms[c.Id].Transition(manager.VmRunning); err != nil {
+		ms.logger.Warn("Failed to transition VM state", "computation", c.Id, "error", err)
```

Comment:
Use the  typed string for "error" here, or a constant

Reply:
it is a log message we don't need a constant for that here